### ISSUE TITLE
[dns] RD bit cannot be unset

### ISSUE THREAD

**Issue by chantra** [RIPE-NCC/ripe-atlas-tools#227]
It seems it is not possible to set RD bit to false using the current version of `ripe-atlas-tools`. This means that there is no way to perform a query against an authoritative/recursive server with RD not set.

AFAICT, the reason seems to be that `set-rd-bit` uses the [`store_true` action](https://docs.python.org/3/library/argparse.html#action), which set the value to True when the argument is used, but default to False otherwise:

> 'store_true' and 'store_false' - These are special cases of 'store_const' used for storing the values True and False respectively. In addition, they create default values of False and True respectively.

Yet, when `set-rd-bit` is defined, it is set to default to `conf["specification"]["types"]["dns"]["set-rd-bit"]`
https://github.com/RIPE-NCC/ripe-atlas-tools/blob/e0606e254fa8c4205261be4eda2334a58b2d0493/ripe/atlas/tools/commands/measure/dns.py#L90-L95

which I understand is `False` by default based on https://github.com/RIPE-NCC/ripe-atlas-tools/blob/e0606e254fa8c4205261be4eda2334a58b2d0493/ripe/atlas/tools/settings/__init__.py#L128

Essentially making RD bit always true with no way to set it false.

In order to keep backward compatibility, it seems one way to solve this would be to use something like:

```
parser.add_argument('--set-rd-bit', dest='rd_bit', action='store_true')
parser.add_argument('--unset-rd-bit', dest='rd_bit', action='store_false')
parser.set_defaults(rd_bit=conf["specification"]["types"]["dns"]["set-rd-bit"])
```

Probably making them [mutually exclusive](https://docs.python.org/3/library/argparse.html#mutual-exclusion) and the right documentation to indicate what is the default.

**Comment by chrisamin** [comment]
Fixed in cfadccf60d09e014f6086058218bb0b7050be9eb for all boolean options where the default is or may be (from user config) True. Will go with the 3.0.0 release.

The naming convention used is "--no-X", so in this case "--no-rd-bit" will prevent the "rd_bit" from being set.

**Comment by chantra** [comment]
Thanks!
